main.go

```diff
@@ -819,11 +937,21 @@ The commands work as following:
 
 	b, _ = arguments.Bool("reboot")
 	if b {
-		err := diagnostics.Reboot(device)
-		if err != nil {
-			log.Error(err)
+		v, err := ios.GetProductVersion(device)
```

Comment:
@dmissmann @diegoperini  I think I want to move this logic to the module. So others can just call reboot without knowing the device version